src/components/Home/ContactMe/ContactMeForm.tsx

```diff
@@ -5,7 +5,17 @@ const ContactMeForm: React.FC = () => {
   const { register, handleSubmit } = useForm();
 
   const onSubmit = (data: any) => {
-    console.log(data);
+    //Attempt to store input data with type assertion
```

Comment:
Can we revert this change and move this a new branch?

Reply:
reverted